vuepress/docs/next/tutorials/create/mfe/epc.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebarDepth: 2
+---
+
+# Create an Entando Platform Capability 
+
+An Entando Platform Capability, or EPC, is a packaged capability in the form of a bundle that adds functionality to the platform such as additional UX controls in the App Builder. With an EPC, adding menu options, an API management page, or integrating a WCMS like Strapi from the App Builder is simplified. This tutorial demonstrates how to build an EPC from a simple React micro frontend bundle.
```

Comment:
@nshaw, can you check that the examples listed make sense.

Reply:
@jyunmitch they work for me

Reply:
"An Entando Platform Capability, or EPC, is a packaged capability in the form of a bundle that adds functionality to the platform" stands on its own as a complete sentence. "such as..." provides an example of added functionality, which supports the statement. we can discuss next style meeting if you think it's important.

Reply:
it's not an example of added functionality, it is the only functionality. So needed to point that out. Removing 'such as' might be the way to go. Will play with it.